### PR TITLE
Archive: Schedules

### PR DESCRIPTION
**Pull request by Mocab**
We should keep this as an archive for a few months, just incase the schedules gets added again.

### CODE REVIEW

**Comment by DaddyChimken** [comment]
I'd go with removing the post completely if its outdated or does not work. You can save the article yourself to a local device.
If for whatever reason the article needs to be added back, then create a PR adding it.

File: _posts/minecraft/bungeecord/2021-06-14-getting-started.md [hunk 1]
```diff
@@ -1,11 +1,11 @@
----
+<!-- ---
 layout: post
 title:  "Getting Started"
 categories: Minecraft
 tags: Bungeecord
 permalink: /minecraft/bungeecord/getting-started/
 ---
-
+-->
```
Comment (DaddyChimken) [inline _posts/minecraft/bungeecord/2021-06-14-getting-started.md:8]: Why?

Comment (Mocab) [reply _posts/minecraft/bungeecord/2021-06-14-getting-started.md:8]: You can ignore that, I tried to turn the metadata into a comment but I failed. In the latest commit the post was completely deleted.

Comment (Mocab) [reply _posts/minecraft/bungeecord/2021-06-14-getting-started.md:8]: Oh my bad, I must have mixed them up.

**Review by DaddyChimken — APPROVED** [review]

**Pull request merged**